### PR TITLE
faq: Amended faq mailing list information

### PR DESCRIPTION
**Pull request by adamsigal**
The previous faq section had outdated information on the mailing list. I amended it with a description and a link.

## Tips:

* please read the style guide before submitting your patch:
[docs/style-guide.md](../docs/style-guide.md)

* commit message titles must be in the form:

```topic: Capitalized message with no trailing period```

or:

```topic, topic2: Capitalized message with no trailing period```

* golang code must be formatted according to the standard, please run:

```
make gofmt		# formats the entire project correctly
```

or format a single golang file correctly:

```
gofmt -w yourcode.go
```

* please rebase your patch against current git master:

```
git checkout master
git pull origin master
git checkout your-feature
git rebase master
git push your-remote your-feature
hub pull-request	# or submit with the github web ui
```

* after a patch review, please ping @purpleidea so we know to re-review:

```
# make changes based on reviews...
git add -p		# add new changes
git commit --amend	# combine with existing commit
git push your-remote your-feature -f
# now ping @purpleidea in the github PR since it doesn't notify us automatically
```

### CODE REVIEW

**Comment by purpleidea** [comment]
Test failed!

**Comment by adamsigal** [comment]
Updated pull request.

File: docs/faq.md [hunk 1]
```diff
@@ -298,9 +298,8 @@ which definitely existed before the band did.
 ### You didn't answer my question, or I have a question!
 
 It's best to ask on [IRC](https://webchat.freenode.net/?channels=#mgmtconfig)
-to see if someone can help you. Once we get a big enough community going, we'll
-add a mailing list. If you don't get any response from the above, you can
-contact me through my [technical blog](https://purpleidea.com/contact/)
+to see if someone can help you. If you don't get any response from the above,
```
Comment (purpleidea) [inline docs/faq.md]: s/the above/IRC/ ?

**Comment by purpleidea** [comment]
LGTM :)

**Comment by purpleidea** [comment]
Merged, thanks!

Welcome to the club :)

**Pull request merged**